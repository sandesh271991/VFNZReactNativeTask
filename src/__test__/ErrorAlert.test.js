/* eslint-disable no-unused-vars */
/* eslint-disable no-undef */

import React from 'react';
import { shallow } from 'enzyme';
import { configure } from 'enzyme';
import __ErrorAlert from '../common/ErrorAlertComponent/errorAlert';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

describe('ErrorAlert', () => {
  it('should render ErrorAlert component', () => {
    const wrapper = shallow(<__ErrorAlert />);
  });

  it('should render initial layout', () => {
    // when
    const component = shallow(<__ErrorAlert />);
    // then
    expect(component.getElements()).toMatchSnapshot();
  });
});
