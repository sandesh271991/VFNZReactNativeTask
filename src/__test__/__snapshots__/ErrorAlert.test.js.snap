// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorAlert should render initial layout 1`] = `
Array [
  <React.Fragment>
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "justifyContent": "center",
        }
      }
    >
      <View
        style={
          Object {
            "backgroundColor": "#f76f6a",
            "borderColor": "red",
            "borderRadius": 10,
            "borderWidth": 2,
            "padding": 10,
          }
        }
      >
        <withTheme(Text)
          style={
            Object {
              "alignItems": "center",
              "borderRadius": 7,
              "color": "white",
              "fontSize": 25,
              "overflow": "hidden",
              "padding": 10,
              "textAlign": "left",
            }
          }
        >
          Something went wrong
        </withTheme(Text)>
        <withTheme(Text)
          style={
            Object {
              "alignItems": "center",
              "borderRadius": 7,
              "color": "white",
              "fontSize": 18,
              "overflow": "hidden",
              "padding": 10,
              "textAlign": "center",
            }
          }
        >
          Please check network connectivity
        </withTheme(Text)>
      </View>
    </View>
  </React.Fragment>,
]
`;
